es-bs-base/scss/utilities/_sizing.scss

```diff
@@ -18,3 +18,22 @@
 
 .vw-100 { width: 100vw !important; }
 .vh-100 { height: 100vh !important; }
+
+// Width Auto Utils
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+    .w#{$infix}-auto { width: auto !important; }
```

Comment:
This one threw me for a loop as I thought `w-lg-auto` was a utility class. Now it is (pending approval) https://github.com/EnergySage/es-ds/blob/main/es-design-system/pages/organisms/es-form.vue#L342-L346